Blish HUD/GameServices/GameIntegration/AudioIntegration.cs

```diff
@@ -42,14 +44,21 @@ public enum Devices {
         private double _timeSinceAudioDeviceUpdate = 0;
 
         private float? _volume;
-
         /// <summary>
         /// This either provides  an estimated volume level for the application
         /// based on the volumes levels exhibited by the game
         /// or
         /// the set volume in settings.
         /// </summary>
-        public float Volume => _volume ??= GetVolume();
+        public float Volume {
+            get => _volume ??= GetVolume();
+            set {
+                if (_volume != value) {
+                    _volume = value;
+                    VolumeChanged?.Invoke(this, new ValueEventArgs<float>(value));
+                }
+            }
```

Comment:
This is currently marked public.  if this is regularly being updated by core, what is the use case to set it outside of core?  Should this be private or internal?

Reply:
First, backward compatibility. 
But also I'd like to have access to Volume outside the event for calculations.
I use this Volume as a master volume for when I setup volume slider settings. I calculate a normalized volume based off of it.

The event is just so I don't need to continuously check the volume in modules (comfortability not having todo that in every module's Update) and can react to a change for sounds still running (eg. music).

Reply:
Oh, I meant the setter specifically which this did not have before.

Reply:
Thanks!